### ISSUE TITLE
Any node can join any node

### ISSUE THREAD

**Issue by akadata** [Syleron/PulseHA#94]
as per the title. any node can join. I would think this is a security issue as any node can join and possibly hijack the network resources.

**Comment by Syleron** [comment]
@akadata This is true if you are not using TLS and hence not of real concern imo as you shouldn't be using PulseHA without TLS over the internet.
